transport.go

```diff
@@ -89,19 +99,26 @@ type WithPeers interface {
 	DisconnectAll()                          // Disconnect all peers, possibly to reconnect them later
 }
 
-// AppendPipeline is used for pipelining AppendEntries requests. It is used
+// AppendPipeline2 is used for pipelining AppendEntries requests. It is used
```

Comment:
The "2" seems gross but I can't come up with anything better right now. I'll revisit this after you merge :-)